src/lib/testFlags.ts

```diff
@@ -43,6 +43,14 @@ class TestFlags {
   get isolatedMargin() {
     return !!this.queryParams.isolatedmargin;
   }
+
+  get displaySocialLogin() {
+    return !!this.queryParams.displaySocialLogin;
```

Comment:
We normalize the flags to be lower case, so we'll need to update the check to:
`return !!this.queryParams.displaysocialogin;`